### ISSUE TITLE
Generate alt text quick fix doesn't work, shows quick pick flashing and closing

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#247634]
Testing #247539

https://github.com/user-attachments/assets/9ec608b2-e337-42bf-a540-01ffdb31d1ba

**Comment by justschen** [comment]
i think same as https://github.com/microsoft/vscode/issues/247632. i'm not seeing this either
